src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -256,6 +265,23 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     # Checking if custom locations rp is registered and fetching oid if it is registered
     enable_custom_locations, custom_locations_oid = check_cl_registration_and_get_oid(cmd, cl_oid)
 
+    try:
```

Comment:
Let's do this check just after the step: '# Check Release Existance'. Reason is, if its first time onboarding, then we should do minimal operations before doing this check. And if its a re-put, we should not block it due to this check